frontend/dockerfile/dockerfile_test.go

```diff
@@ -6562,7 +6562,10 @@ FROM scratch
 COPY --from=0 / /
 `)
 
-	const expectedDigest = "sha256:9e36395384d073e711102b13bd0ba4b779ef6afbaf5cadeb77fe77dba8967d1f"
+	// note that this digest differs from the one in master, due to
+	// commit a89f482dcb3428c0297f39474eebd7de15e4792a not being included
+	// in this branch.
+	const expectedDigest = "sha256:e26093cc8a7524089a1d0136457e6c09a34176e2b2efcf99ac471baa729c7dc9"
```

Comment:
Added a comment here; we'll have to update this test if we were to decide to backport that commit;

- https://github.com/moby/buildkit/commit/a89f482dcb3428c0297f39474eebd7de15e4792a (https://github.com/moby/buildkit/pull/3600)